demos/embeddings/deep-graph-infomax-cora.ipynb

```diff
@@ -6,7 +6,7 @@
    "source": [
```

Comment:
<p>This isn't part of this PR directly (and so doesn't need to be fixed here), but it looks like this number is pretty close to the supervised task: https://github.com/stellargraph/stellargraph/blob/develop/demos/node-classification/gcn/gcn-cora-node-classification-example.ipynb</p><p> </p><p>Maybe we could mention this; something like (please edit!)</p><pre class="ql-syntax" spellcheck="false">This accuracy is close to that for [training a supervised GCN model end-to-end](../node-classification/gcn/gcn-cora-node-classification-example.ipynb), suggesting that Deep Graph Infomax is an effective method for unsupervised training.
</pre><p>(Note the relative link is important, so that it works in jupyterlab/colab etc.)</p><p> </p><p>Of course the setup is a bit different here; this has a only a single GCN layer with a much large embedding size, vs the two 16-dim GCN layers in the supervised one.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/stellargraph/stellargraph/pull/1162/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
great idea for the relevant link! I didn't edit in the end bc I think your comment states that nicely :)